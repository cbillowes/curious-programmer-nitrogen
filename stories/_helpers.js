import React from 'react'
import Layout from '../src/components/layout'
import { SmallSpacer } from '../src/components/spacer'

const Page = ({ className, children }) => {
  return (
    <Layout
      className={className}
    >
      <SmallSpacer />
      {children}
    </Layout>
  )
}

export default Page