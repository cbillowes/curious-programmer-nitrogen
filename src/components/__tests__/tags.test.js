import React from 'react'
import Tags from '../tags'
import { getSnapshot } from './_helpers'

describe(`Tags`, () => {
  it(`should render`, () => {
    const tags = (
      <Tags />
    )
    const tree = getSnapshot(tags)
    expect(tree).toMatchSnapshot()
  })
})
