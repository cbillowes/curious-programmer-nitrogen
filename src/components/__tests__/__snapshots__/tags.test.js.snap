// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags should render from list 1`] = `
<div
  className="tags"
>
   
  <a
    className="tag"
    href="/tag/hello/"
    onClick={[Function]}
    onMouseEnter={[Function]}
    title="hello"
  >
    hello
  </a>
   
   
  <a
    className="tag"
    href="/tag/world/"
    onClick={[Function]}
    onMouseEnter={[Function]}
    title="world"
  >
    world
  </a>
   
</div>
`;
